.storybook/webpack.config.js

```diff
@@ -55,9 +55,22 @@ module.exports = async ( { config } ) => {
 		},
 	);
 
+	// exclude existing svg rule created by storybook before pushing custom rule
+	const fileLoaderRule = config.module.rules.find( ( rule ) => rule.test && rule.test.test( '.svg' ) );
+	fileLoaderRule.exclude = /\.svg$/;
```

Comment:
Why would we exclude the rule? Maybe we should just remove it so we only use the one going forward? Just curious if there is a reason why we would want/need it now that we're doing the same outside of storybook.

Reply:
Storybook automatically adds a rule for SVGs in the background that conflicts with the rule we try to use with our SVGR config. This is required for SVGs within Storybook to match behavior within our standard application.